<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelprint_goodinfo_select_dialog</class>
 <widget class="QDialog" name="labelprint_goodinfo_select_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="outer_stylesheet" stdset="0">
   <string>dialog_hasborder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="label_explain">
       <property name="styleSheet">
        <string notr="true">color:blue;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/ok.png</normaloff>:/image/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button_nowrap" name="pushButton_2">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="peripheral.qrc">
         <normaloff>:/image/exit.png</normaloff>:/image/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button_nowrap</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button_nowrap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="peripheral.qrc"/>
 </resources>
 <connections/>
</ui>
